docker_release/main.py

```diff
@@ -207,9 +207,9 @@ def main():
             image = '%s/%s:latest' % (organization, repository)
             _docker_build(docker, args, docker_image_dir, image)
 
-            latest_tag = 'latest-snapshot'
```

Comment:
I would keep latest-snapshot, as in that way I would need to change the version (tag) each time I push a snapshot in case of development testing.
I would prefer to keep that over git commit hash.